src/neuroconv/schemas/base_metadata_schema.json

```diff
@@ -0,0 +1,171 @@
+          "type": "array",
+          "items": {
+            "type": "string",
+            "title": "experimenter"
+          }
+        },
+        "identifier": {
+          "type": "string",
+          "description": "A unique text identifier for the file. If one is not provided it will be randomly assigned"
+        },
+        "institution": {
+          "type": "string",
+          "description": "Institution(s) where experiment is performed"
+        },
+        "lab": {
+          "type": "string",
+          "description": "Lab where experiment was performed"
+        },
+        "session_description": {
+          "type": "string",
+          "description": "A description of the session where this data was generated"
+        },
+        "session_start_time": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The start date and time of the recording session"
+        },
+        "surgery": {
+          "type": "string",
+          "description": "Narrative description about surgery/surgeries, including date(s) and who performed surgery."
+        },
+        "pharmacology": {
+          "type": "string",
+          "description": "Description of drugs used, including how and when they were administered. Anesthesia(s), painkiller(s), etc., plus dosage, concentration, etc."
+        },
+        "protocol": {
+          "type": "string",
+          "description": "Experimental protocol, if applicable. E.g., include IACUC protocol"
+        },
+        "related_publications": {
+          "type": "array",
+          "items": {
+            "title": "related publication",
+            "type": "string"
+          }
+        },
+        "slices": {
+          "type": "string",
+          "description": "Description of slices, including information about preparation thickness, orientation, temperature and bath solution."
+        },
+        "source_script": {
+          "type": "string",
+          "description": "Script file used to create this NWB file."
+        },
+        "source_script_file_name": {
+          "type": "string",
+          "description": "Script file used to create this NWB file."
+        },
+        "notes": {
+          "type": "string",
+          "description": "Notes about the experiment."
+        },
+        "virus": {
+          "type": "string",
+          "description": "Narrative description about surgery/surgeries, including date(s) and who performed surgery."
+        },
+        "data_collection": {
+          "type": "string",
+          "description": "Notes about data collection and analysis."
+        },
+        "stimulus_notes": {
+          "type": "string",
+          "description": "Notes about data collection and analysis."
+        }
+      }
+    },
+    "Subject": {
+      "type": "object",
+      "required": [
+        "subject_id",
+        "sex",
+        "species"
+      ],
```

Comment:
I know this gets a little tricky with the 'either/or' logic with `age` _or_ `date_of_birth` - would it be totally out there to consider _both_ as requirements and auto-fill the other when one is specified?

Not for NWB files in general of course, just through NeuroConv/GUIDE automation